web/admin/src/request/Crawler.ts

```diff
@@ -106,21 +112,21 @@ export const postApiV1CrawlerEpubConvert = (
  *
  * @tags crawler
  * @name PostApiV1CrawlerFeishuGetDoc
- * @summary FeishuGetDocx
+ * @summary FeishuDoc
  * @request POST:/api/v1/crawler/feishu/get_doc
  * @response `200` `(DomainPWResponse & {
-    data?: (DomainGetDocxResp)[],
+    data?: (V1FeishuGetDocResp)[],
 
 })` OK
  */
 
 export const postApiV1CrawlerFeishuGetDoc = (
-  body: DomainGetDocxReq,
+  body: V1FeishuGetDocReq,
```

Comment:
在Crawler.ts文件中，API接口定义可以进一步完善以提高类型安全性。

```suggestion
// 建议为返回的数据添加更明确的类型定义
export const postApiV1CrawlerFeishuGetDoc = (
  body: V1FeishuGetDocReq,
  params: RequestParams = {},
) =>
  httpRequest<
    DomainPWResponse & {
      // 明确指定data为非空数组
      data: V1FeishuGetDocResp[];
    }
  >({
    path: `/api/v1/crawler/feishu/get_doc`,
    method: "POST",
    body: body,
    type: ContentType.Json,
    format: "json",
    ...params,
  });